GAE/src/org/akvo/flow/rest/dto/SurveyAssignmentDto.java

```diff
@@ -0,0 +1,92 @@
+/*
+ *  Copyright (C) 2019 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.flow.rest.dto;
+
+import java.util.Date;
+import java.util.List;
+
+import com.gallatinsystems.framework.gwt.dto.client.BaseDto;
+
+public class SurveyAssignmentDto extends BaseDto {
```

Comment:
I thought there was already a `SurveyAssignmentDto` class..? Isnt this duplicating it?

Reply:
Moving.